pof/src/types.rs

```diff
@@ -2423,7 +2461,9 @@ pub enum Warning {
     SpecialPointPropertiesTooLong(usize),
     // path with no parent
     // thruster with no engine subsys (and an engine subsys exists)
-    // turret uvec != turret normal
```

Comment:
`turret uvec != turret normal` is still a valid concern here. You don't have to add it, but leave the reminder comment.